docs/prebuilt.md

```diff
@@ -0,0 +1,32 @@
+# Prebuilt Binaries
+
+There are some prebuilt binaries placed in [rv32emu-prebuilt](https://github.com/sysprog21/rv32emu-prebuilt). When invoking testing or benchmarking, the prebuilt binaries will be pulled into `build/linux-x64/` and `build/riscv32/` directory in default.
+
+To fetch the prebuilt binaries manually, run:
+
+```shell
+$ make artifact
+```
+
+Or build the binaries from scratch (the RISC-V cross-compiler is required):
+
+```shell
+$ make artifact USE_PREBUILT=0 [CROSS_COMPILE=<COMPILER_PREFIX>]
+```
+
+The compiler prefix varies according to the used toolchain, such as `riscv-none-elf-`, `riscv32-unknwon-elf-`, etc.
+
+---
+
+There are still some standalone RISC-V binaries under `build/` directory for testing purpose.
```

Comment:
You should mention the following:
* uaes
* captcha
* cc
* chacha20
* donut
* fcalc
* hamilton
* jit-bf
* lena
* line
* maj2random
* mandelbrot
* nqueens
* nyancat
* pi
* qrcode
* readelf
* richards
* rvsim
* smolnes
* spirograph

And then make a list of the benchmark programs provided by rv8-bench and ansibench.